src/axum_websocket.rs

```diff
@@ -0,0 +1,883 @@
+#![allow(unused)]
+
+//! Modified from https://github.com/tokio-rs/axum/blob/main/axum/src/extract/ws.rs
```

Comment:
Is this disclaimer enough to indicate that we have modified a small part of the Axum implementation of WebSocket? Any license that needs to be mentioned here?

Reply:
I'd use a permalink instead. The `main` branch will change.

If we only need this file from Axum: I'd copy the license text into this file.
I you think we'll need to patch more, we can mention it in our README or LICENSE file.